core/pom.xml

```diff
@@ -267,6 +267,25 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>3.1.1</version>
```

Comment:
Nit: the plugin version is already defined by the `<pluginManagement>` section of the parent POM, could you remove it here? That will make things easier if we need to upgrade it in the future.

Reply:
done